src/js/step.js

```diff
@@ -357,7 +353,8 @@ export class Step extends Evented {
         arrow: true
       },
       tourOptions,
-      options
+      options,
+      mergeTooltipConfig(tourOptions, options)
```

Comment:
same as setupTooltip/destroyTooltip, moved that code to it's own file so we can change all floatingUI related options in one file.